plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -81,14 +83,25 @@ def stop_watching
       end
     end
 
-    def client
-      deploy_group.kubernetes_cluster.client
-    end
-
     def docs_by_role
       @docs_by_role ||= release_docs.each_with_object({}) do |rel_doc, hash|
         hash[rel_doc.kubernetes_role.label_name] = rel_doc
       end
     end
+
+    def deploy_group_ids
+      release_docs.map(&:deploy_group_id)
+    end
+
+    def deploy_group_ids=(id_list)
```

Comment:
Nope, we aren't using it anymore. The controller was using this mechanism to create releases under the release_group.

I couldn't figure out how to get rid of this function as the permit params on the controller was not liking me not using the passed in params.